src/viewer.js

```diff
@@ -107,12 +111,18 @@ class Viewer {
   createInitParams_() {
     const parsedViewerUrl = parseUrl(window.location.href);
 
-    // TODO (chenshay): set more init params.
     const initParams = {
       origin: parsedViewerUrl.origin,
     };
 
     if (this.referrer_) initParams.referrer = this.referrer_;
+    if (this.prerender_) {
+      initParams.visibilityState = 'prerender';
+      initParams.prerenderSize = 1;
```

Comment:
Do these need to be quoted so they're not obfuscated by the compiler? Or do you have externs?